pkg/node/node.go

```diff
@@ -944,7 +945,7 @@ func NewBee(interrupt chan struct{}, sysInterrupt chan os.Signal, addr string, p
 	pushSyncProtocolSpec := pushSyncProtocol.Protocol()
 	pullSyncProtocolSpec := pullSyncProtocol.Protocol()
 
-	if o.FullNodeMode {
+	if o.FullNodeMode && !o.BootnodeMode {
```

Comment:
this may be the only change needed, the  `includeSelf` changes may not be needed